### ISSUE TITLE
[Question C#] Update Nuget 3.5.8 PromptDialog.Choice + SuggestedActions on Skype

### ISSUE THREAD

**Issue by Noc2** [Microsoft/BotBuilder#2706]
Probably I am just impatient. Sorry :-) Since my bot causes some random channel failures on Skype (the new context.UserData.GetValue might help) I like to test the new nuget version, but therefore PromptDialog.Choice needs to work (#2685). 

So, here are my questions:
- When are you planning to release a fixed version?
- Will Skype support SuggestedActions? (I think quick replies are really useful)  

Btw thanks for your work. You are doing a great job 👍.

**Comment by nickcordrey** [comment]
You need to be on a recent version of Skype - https://web.skype.com/ supports suggested actions so please test there and let us know if you still see an issue.

**Comment by Noc2** [comment]
So SuggestedActions are already implemented on Skype? Cool!
But currently they aren’t working for me? Probably, it’s an issue with my code:
```
IMessageActivity reply = context.MakeMessage();
List<CardAction> cardButtons = new List<CardAction>();
CardAction button1 = new CardAction()
{
Type = "imBack",
Title = buttonText1,
Value = buttonText1
};
CardAction button2 = new CardAction()
{
Type = "imBack",
Title = buttonText2,
Value = buttonText2
};
CardAction button3 = new CardAction()
{
Type = "imBack",
Title = buttonText3,
Value = buttonText3
};
CardAction button4 = new CardAction()
{
Type = "imBack",
Title = buttonText4,
Value = buttonText4
};
CardAction button5 = new CardAction()
{
Type = "imBack",
Title = "Cancel ❌",
Value = "Cancel ❌",
};
cardButtons.Add(button1);
cardButtons.Add(button2);
cardButtons.Add(button3);
cardButtons.Add(button4);
cardButtons.Add(button5);
reply.Text = text;
reply.SuggestedActions = new SuggestedActions(actions: cardButtons);
```

**Comment by ejadib** [comment]
FYI - A new package with fix #2685 was just published (https://www.nuget.org/packages/Microsoft.Bot.Builder/3.5.9)

**Comment by Noc2** [comment]
I opened a new issue #2712, since SuggestedActions still don’t work for me on Skype.
